### ISSUE TITLE
Yielding part of float numbers/prices

### ISSUE THREAD

**Issue by olgakuak** [KoljaB/stream2sentence#2]
Hello! Using the "quick_yield_single_sentence_fragment" parameter entails the separation of non-integer numbers and prices, which significantly affects the quality of the TTS:
["The price for Pure Leaf Sweet Tea is $3.", "5 for small size and $5 for large size."]

**Comment by KoljaB** [comment]
Yes, true. I use the NLTK or stanza tokenizers for the standard "splitting into sentence" logic. These tokenizers can't split in comma as of my knowledge. Therefore quick_yield_single_sentence_fragment uses a very simplified splitting logic only based on detection of single characters. 

I am open to any suggestions how to enhance this.

**Comment by KoljaB** [comment]
Fixed in v1.0.0 now
